downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md

```diff
@@ -144,6 +144,22 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
            sample:main(sample.bal:3)
     ```
 
+- Fixed a bug that resulted in a compiler crash when attempting to ignore the error cause using `_` in an error binding pattern.
+
+  ```ballerina
+  type SampleErrorData record {|
+    int code;
+    string reason;
+  |};
+
+  type SampleError error<SampleErrorData>;
+
+  public function main() {
+    SampleError sampleErr = error("Transaction Failure", error("Database Error"), code = 20, reason = "deadlock condition");
```

Comment:
Formatting (tab) seems off. Let's also simplify the example, IMO we don't need two detail fields here.

Reply:
May not even need a custom error.